src/node_file.cc

```diff
@@ -3373,6 +3391,7 @@ static void CreatePerIsolateProperties(IsolateData* isolate_data,
   SetMethod(isolate, target, "readBuffers", ReadBuffers);
   SetMethod(isolate, target, "fdatasync", Fdatasync);
   SetMethod(isolate, target, "fsync", Fsync);
+  SetMethod(isolate, target, "fsyncSync", FsyncSync);
```

Comment:
Can you also update `typings/internalBindings/fs.d.ts` file with this new function?